after_success

```diff
@@ -113,7 +113,9 @@ elif [[ ${CI_OS_NAME} = linux ]]; then
       exit 0
     fi
 
-    generate_coverage_data
+    if [[ ";${DO_COVERAGE_ON_BRANCH};" == *";${TRAVIS_BRANCH};"* ]]; then
```

Comment:
Should be `CI_BRANCH` instead of `TRAVIS_BRANCH` (idem for the 2 other occurrences)

Also, what is the role of the `*` around the right operand?